components/table/ApplicationTable.tsx

```diff
@@ -140,19 +149,36 @@ const ApplicationTable: FC<ApplicationTableProps> = ({
         header: 'Reviewer',
         id: SEARCH_PARAM_REVIEWER
       }),
+      columnHelper.accessor('internalReview.reviewerPercentile', {
+        cell: (info) => info.getValue() || 'N/A',
+        header: 'R %',
+        id: 'percentile',
+        filterFn: (row) => {
+          const value = row.original.internalReview?.reviewerPercentile
+          return (
+            value !== undefined &&
+            value !== null &&
```

Comment:
You can use lodash's `isNil`

```suggestion
           !isNil(value) &&
```